addon/services/street.js

```diff
@@ -16,6 +17,10 @@ export default class StreetService extends GWRService {
     return this.cache(street);
   }
 
+  get language() {
+    return languageOptions[this.intl.primaryLocale?.split("-")?.[0]];
```

Comment:
In other places we just use `this.intl.primaryLocale` without the split. Could this pose a problem for the other language parts?

Reply:
I only found one other location with the `this.intl.primaryLocale` for date input fields and there luxon can handle both "de" and "de-ch" formatted locales. The problem only arises for the street language because we have a language options object defined, that expects the locale to be in the format "de", "fr" or "it".

Reply:
ok, thanks for clarifying!